HW02_A_Graph-Fails.Rmd

```diff
@@ -33,55 +33,79 @@ What error is being thrown? How do you correct it? (hint, the error message tell
 data(mpg) #this is a dataset from the ggplot2 package
 
 mpg %>% 
-  ggplot(mapping = aes(x = city, y = hwy, color = "blue")) %>% 
+  ggplot(mapping = aes(x = city, y = hwy , color = "blue")) + 
   geom_point()
 
 ```
+The initial error was the %% connecting the ggplot() and the geom_point() functions. After I changed the %% to a +, I keep getting an error message saying city is not found. I look at the column headings in the dataset, city is labeled as cty, and still I cannot get the plot to show up (despite changing the x-axis to "x = cty) I am leaving this alone. - DS
+
 
 ### Graph Fail 2
 Why aren't the points blue? It is making me blue that the points in the graph aren't blue :`(
 ```{r}
 ggplot(data = mpg) + 
-  geom_point(mapping = aes(x = displ, y = hwy, color = "blue"))
+  geom_point(mapping = aes(x = displ, y = hwy), color = "blue")
 
 ```
+I've made the color equal the hexadecimal system R uses for color. I used 000000, even though I believe that is black. But it looks blue here. I am not sure why the color argument is the name of the legend. but yolo.- DS
 
+Now I see, I had the color assignment within aesthetics. to assign color, it needs to be outside of that function. I have since corrected it. - DS
 
 
 ### Graph Fail 3
 Two mistakes in this graph. First, I wanted to make the the points slightly bolder, but changing the alpha to 2 does nothing. What does alpha do and what does setting it to 2 do? What could be done instead if I want the points slightly bigger? 
 
 Second, I wanted to move the legend on top of the graph since there aren't any points there, putting it at approximately the point/ordered pair (5, 40). How do you actually do this? Also, how do you remove the legend title ("class")? Finally, how would you remove the plot legend completely?
 ```{r}
-mpg %>% 
-ggplot() + 
-  geom_point(mapping = aes(x = displ, y = hwy, color = class), alpha = 2) + 
-  theme(legend.direction = "horizontal") + 
-  theme(legend.position = c(5, 40))
+p <- mpg %>% 
+ ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), alpha = 2, size = 2) 
+
+q <- p + theme(legend.position = c(0.6, 0.85)) + theme(legend.direction = "horizontal")
+
+##Removing the legend title
+q + theme(legend.title = element_blank())
 
+##Removing the plot legend
+## q + theme(legend.position = 'none')
 ```

```

Comment:
I like how you assigned the plots to a letter so that you didn't have to include another block of repetitive code (...like I did) when answering the different question prompts